lib/rucio/rse/protocols/protocol.py

```diff
@@ -41,16 +41,31 @@ class RSEDeterministicScopeTranslation(PolicyPackageAlgorithms):
     """
         Translates a pfn dictionary into a scope and name
     """
+
+    _algorithm_type = "pfn2scope_name"
```

Comment:
I think, `pfn2lfn` would be a more standard name. 
Since this is supposed to be an inverse operation of `lfn2pfn` algorithms.

It is understood that `scope` is implicitly an output and input in the `pfn2lfn` and `lfn2pfn` policy package functions respectively.

Reply:
okay, I don't have an objection to this. I'll do all the changes at once once the other thread is resolved.

Reply:
I renamed it, does it look good now?